torch/_dynamo/variables/nn_module.py

```diff
@@ -686,9 +687,15 @@ def call_function(
         else:
             source = None
 
-        return variables.UserFunctionVariable(
-            fn, source=source, **options
-        ).call_function(tx, [self] + list(args), kwargs)
+        ctx = (
+            record_nn_module_stack(str(id(mod)), self.source, tx, mod)
+            if self.source
+            else nullcontext()
+        )
+        with ctx:
+            return variables.UserFunctionVariable(
+                fn, source=source, **options
+            ).call_function(tx, [self] + list(args), kwargs)
```

Comment:
Can add a dynamo test in ```dynamo/test_module.py``` to check if it works well when this inlining failed. Otherwise, this looks good!

Reply:
np, will do that!